harness_cloud_cost_management/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3eb2e9ef-2c9c-45b6-8f1c-8a900910f948",
+  "app_id": "harness-cloud-cost-management",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "description": "Cloud Cost Management allows Harness customers to access and view their cloud and cluster cost metrics over a specified range of dates",
+    "title": "Harness Cloud Cost Management",
```

Comment:
```suggestion
    "title": "Harness Cloud Cost Mgmt",
```

Reply:
Hi @mgarabed 

Thanks for the suggestion! Could you please explain why do we need to change 'Management' to 'Mgmt'? Is this text shown anywhere in the UI?